requirements.txt

```diff
@@ -12,4 +12,5 @@ transformers>=4.31.0
 sentencepiece
 py7zr
 scipy
-optimum
\ No newline at end of file
+optimum
+aim=3.7.11
```

Comment:
probably want to make this optional. I think we're going to want to ensure that users that don't have (don't want) aim installed should still be able to use llama-recipes.